openquake/hazardlib/geo/surface/base.py

```diff
@@ -427,6 +427,41 @@ def get_middle_point(self):
 
         return mesh.get_middle_point()
 
+    def get_resampled_top_edge(self, angle_var=0.1):
```

Comment:
This method, if I'm not mistaken appears also in the pull request #292 https://github.com/gem/oq-hazardlib/pull/292/files#diff-5a3586c46e08bf2621d8adfa405017fbR430